requirements/requirements.txt

```diff
@@ -0,0 +1,2169 @@
+    # via requests
+click==8.1.8 \
+    --hash=sha256:63c132bbbed01578a06712a2d1f497bb62d9c1c0d329b7903a866228027263b2 \
+    --hash=sha256:ed53c9d8990d83c2a27deae68e4ee337473f6330c040a31d4225c9574d16096a
+    # via
+    #   dask
+    #   distributed
+    #   flask
+    #   mlflow-skinny
+    #   uvicorn
+cliff==4.9.1 \
+    --hash=sha256:3e5861dcc56164b7c0aef2dae166b537c2dc11c9df785f74f54bdff16971fb1e \
+    --hash=sha256:5b392198293c0b9225d459be8ba710cf8248f1ee33006bdeb3d92fb0012592b4
+    # via optuna
+cloudpickle==3.1.1 \
+    --hash=sha256:b216fa8ae4019d5482a8ac3c95d8f6346115d8835911fd4aefd1a445e4242c64 \
+    --hash=sha256:c8c5a44295039331ee9dad40ba100a9c7297b6f988e50e87ccdf3765a668350e
+    # via
+    #   dask
+    #   distributed
+    #   mlflow-skinny
+cmaes==0.11.1 \
+    --hash=sha256:1de77d2175045389680619c1e9b6d59d90e7888524d9e440e1704ba001de9fcc \
+    --hash=sha256:cf71fa3679814723be771f2c9edd85f465b1bc1e409e1ad6d8a9e481efcd5160
+    # via optuna
+cmake==4.0.0 \
+    --hash=sha256:02dd7305ed88d9c98930116fa66b327034e01080b6688886d478b099bd6bf7ba \
+    --hash=sha256:098ceee569eedc7853792f8b4fc6cc5b2995c1481b8bc13074aa565c2ebbac0d \
+    --hash=sha256:26d0e992eea03d7ad8a2f320884f8eaf8178c3cf23f2f5f004cdca8354adf137 \
+    --hash=sha256:2877636e057a8227b1792614bb0eebab37632c73bae3b4939b2b20416248cf76 \
+    --hash=sha256:2f01499980f5824092c08e8fe0893c31d4f3783c8475ea2d7cd0706c7dc646a3 \
+    --hash=sha256:31c4a1a3490f142c965de644e65987d1c76a2b90dec71a9a001c85bf535a5e58 \
+    --hash=sha256:340558bf3b50876380ac036d9e8a0e8c30ef28b071097cbceb3929519c021d4a \
+    --hash=sha256:4b2e75474ee412ca6f7f224b14a883206adc423daff1745752ce815cc5fbb599 \
+    --hash=sha256:5dda13b113de7dba00f20587011c1b9b90708a22fe8fef530a46bfb4a4ee2bd2 \
+    --hash=sha256:6a89cf41a770763a2132b32514dfd8000e1147ecec8cb5ad3d5d83041faea790 \
+    --hash=sha256:8606d0228529d9cb688fc8e4e31ae14236526fad586680aa3e15f6dd69c76488 \
+    --hash=sha256:9af3ef2931c84557d58383169cc3cad6852de625c1fd8883ee696ac436ab1eb3 \
+    --hash=sha256:a7ec8b997f75da5310c571f79ce560bc8941013549e47144bfa49025ced79b00 \
+    --hash=sha256:a9b7b1da83219d563cae0685c989d8aedf58480de1e64f3de2f51364606272f0 \
+    --hash=sha256:b929ab7d2ebd6b9a81018b0248ea75edb3dc4ad69367e8c75fb0f6774bb6e962 \
+    --hash=sha256:ba414b29459526bb10be13ecd38d022f0b5ebed2fec33bdae66d1568ddcf2e2e \
+    --hash=sha256:e1092881c07c5c892448aad04bdc36357804f37ad2ff42272fd1a940f679aa1a \
+    --hash=sha256:e27776fbb5a101a9c8b71f9f360918d0985dfd7d1f057fa90713f8f2125e0e73 \
+    --hash=sha256:f23eaade0cf683c938849962c09b3c752cf96cddc872288c80620466e6acf0ce
+    # via
+    #   pylibczirw
+    #   triton
+cmd2==2.5.11 \
+    --hash=sha256:30a0d385021fbe4a4116672845e5695bbe56eb682f9096066776394f954a7429 \
+    --hash=sha256:cbc79525e423dc2085ef7922cdc5586d1fedaecb768cdfb05e5482ee0740b755
+    # via cliff
+colorama==0.4.6 ; sys_platform == 'win32' \
```

Comment:
Example of using `sys_platform` conditional to make requirements.txt cross-platform